tasks/shelter/README.md

```diff
@@ -296,334 +296,334 @@ Pay attention to additions:
-- **Нарушение размеров** - это ситуация, когда размеры элементов (кнопок, картинок, карточек животных, иконок) отличаются от заданного на макете более чем на 10 пикселей по высоте или ширине. Проверку проводим таким же образом, как и при нарушении отступов, и если отличия большие, снимаем указанное количество баллов.
-- Длину текста и слов по ширине, а также порядок переноса слов по строкам, мы **не учитываем при проверке**.
-- **Уникальная ошибка** - это новая найденная ошибка в проверяемом блоке. Независимо от того, сколько раз эта ошибка повторяется в проверяемом блоке, указанное количество баллов снимается только 1 раз.  
-  Например, съехала кнопка во всех карточках животных. Снимать баллы будем лишь один раз, т.к. скорее всего сломан общий стиль. Если карточка со съехавшей кнопкой одна (а в остальных кнопка находится на своем месте) - все равно снимаем указанное количество баллов. Т.е. нет разницы, ошибка в одном повторяющемся элементе, или в нескольких.
+- A **padding violation** is a situation where the distance between elements, or the distance from an element to the edge of the container it is contained in, differs from the one set on the layout by more than 10 pixels in height or width.
+  Consider the situation using the `our friends` section as an example. You will most likely know the approximate distance between elements by eye. However, you are confused by the indentation of the "left" button from the edge of the screen on the left. The distance from the "left" button to the left border of the block is 40px on a screen width of 1280px. So, setting the width to 1280px, we look at the distance. If it differs by up to 10px inclusive (for example, 50px or 30px) - do not remove points. However, if the indents are greater than these values ​​(for example, 51px or 29px), we remove the specified number of points.
+- **Size Violation** is a situation when the dimensions of elements (buttons, pictures, animal cards, icons) differ from those specified on the layout by more than 10 pixels in height or width. The check is carried out in the same way as in case of violation of indents, and if the differences are large, we remove the specified number of points.
+- The length of the text and words in width, as well as the order of word wrapping in lines, we **do not take into account when checking**.
+- **Unique error** is the new error found in the block being checked. Regardless of how many times this error is repeated in the block being checked, the specified number of points is deducted only 1 time.
+  For example, the button in all animal cards moved out. We will remove points only once, because most likely the general style is broken. If there is only one card with the button that has moved down (and in the rest the button is in its place), we still remove the indicated number of points. Those, there is no difference, the error is in one repeated element, or in several.
```

Comment:
```suggestion
  For example, the button in all animal cards moved out. We will remove points only once because most likely the general style is broken. If there is only one card with the button that has moved down (and in the rest, the button is in its place), we still remove the indicated number of points, i.e. there is no difference, the error is in one repeated element, or in several.
```